src/reactors.js

```diff
@@ -45,16 +45,23 @@ export let reactors = {
   // ported from main app ¯\_(ツ)_/¯
   field: standardChange,
   type: standardChange,
-  mutate: (parent, node, event, reactor, types, lookup) =>
-    _.flow(
+  mutate(parent, node, event, reactor, types, lookup) {
+    if (
+      !hasValue(event.node) &&
+      !hasValue(event.previous) &&
+      !hasContext(event.node)
```

Comment:
this check should be removed - nodes with contexts can and often do trigger other nodes to update (for example `facets`).

Reply:
Lots of tests are broken without it, what if a mutation happens inside context but hasValue is false? do we want to cancel that tree update?